### PR TITLE
Portfolio-setup-and-mobile-version-skeleton

### PR DESCRIPTION
**Pull request by er-contreras**
In this PR we add our about section and work section using grid and flexbox. We include icons and some screenshots.

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -81,9 +75,126 @@ p {
...
-  margin: 4rem 2rem;
+  margin: 1rem 2rem;
   color: gray;
   border: none;
   height: 1px;
   background-color: rgb(72, 72, 72);
 }
+
+.links {
+  /* justify-content: space-between; */
```
Comment (Shaher-11) [inline style.css]: Remove the comment here please

File: index.html [hunk 2]
```diff
@@ -15,17 +15,190 @@ <h2 class="flex"><a href="#" target="_blank">John Doe</a></h2>
...
   <hr>
+
+  <Section class="links flex" >
+    <div class="language-links">
+      <h3>Language</h3>
+
+      <p>Javascript</p>
+      <p>Ruby</p>
+      <p>Html</p>
+      <p>Css</p>
```
Comment (Shaher-11) [inline index.html]: Replace the p tags with ul tags because the usage of p tag meant to be only for paragraphs

File: index.html [hunk 3]
```diff
@@ -15,17 +15,190 @@ <h2 class="flex"><a href="#" target="_blank">John Doe</a></h2>
...
+    </div>
+    
+    <div class="frameworks-links">
+      <h3>Frameworks</h3>
+
+      <p>Bootstrap</p>
+      <p>Ruby on Rails</p>
+      <p>RSPec</p>
+      <p>CapyBara</p>
+      <p>Selenium</p>
```
Comment (Shaher-11) [inline index.html]: Replace the p tags with ul tags because the usage of p tag meant to be only for paragraphs

File: index.html [hunk 4]
```diff
@@ -15,17 +15,190 @@ <h2 class="flex"><a href="#" target="_blank">John Doe</a></h2>
...
+    </div>
+    
+    <div class="skills-links">
+      <h3>Skills</h3>
+
+      <p>Codekit</p>
+      <p>Github</p>
+      <p>Codepen</p>
+      <p>Gitlab</p>
+      <p>Terminal</p>
```
Comment (Shaher-11) [inline index.html]: Replace the p tags with ul tags because the usage of p tag meant to be only for paragraphs

File: index.html [hunk 5]
```diff
@@ -15,17 +15,190 @@ <h2 class="flex"><a href="#" target="_blank">John Doe</a></h2>
...
+      <a href="#"><img src="/icons/github.png" alt="github"></a>
+    </div>
+
+    <div>
+      <img src="/icons/in.png" alt="in">
+    </div>
+
+    <div>
+      <img src="/icons/2.png" alt="2">
+    </div>
```
Comment (Shaher-11) [inline index.html]: the alt attribute should describe what is the image

File: index.html [hunk 6]
```diff
@@ -15,17 +15,190 @@ <h2 class="flex"><a href="#" target="_blank">John Doe</a></h2>
...
+
+    <div>
+      <img src="/icons/twitter.png" alt="twitter">
+    </div>
+
+    <div>
+      <img src="/icons/M.png" alt="m">
+    </div>
```
Comment (Shaher-11) [inline index.html]: the alt attribute should describe what is the image

File: style.css [hunk 7]
```diff
@@ -81,9 +75,126 @@ p {
...
+
+.vertical-lines p {
+  color: white;
+  font-size: 1.5rem;
+  justify-content: space-between;
+}
+
+.vl {
+  border-left: 1.5px solid rgb(62, 62, 62);
+  height: 30px;
```
Comment (Shaher-11) [inline style.css]: a descriptive class name will be very helpful and it is a good practice

**Review by Shaher-11 — CHANGES_REQUESTED** [review]
Hello @eri8-9 

#### Great job so far!
You have been doing an incredible  job on the project so far:clap::clap:
The project is very well structured:blush::blush:
However, we still need to make a few changes to get the project approved:muscle:

 ## Required Changes ♻️

- [x] The skills and languages and frameworks should be in a list.
- [x] Remove the comment from your code.
- [x] Add a screenshot of your project to the readme file.

## Optional Suggestions 💡
_The following point is  highly recommend to improve your code quality, Please consider implementing it

- class names should be descriptive as it's a good practice.

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Cheers and Happy coding!👏👏👏

**Comment by er-contreras** [comment]
> Hello @eri8-9
> 
> #### Great job so far!
> You have been doing an incredible job on the project so far👏👏
> The project is very well structured😊😊
> However, we still need to make a few changes to get the project approved💪
> 
> ## Required Changes ♻️
> * [ ]  The skills and languages and frameworks should be in a list.
> * [ ]  Remove the comment from your code.
> * [ ]  Add a screenshot of your project to the readme file.
> 
> ## Optional Suggestions 💡
> _The following point is highly recommended to improve your code quality, Please consider implementing it
> 
> * class names should be descriptive as it's a good practice.
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
> Cheers and Happy coding!👏👏👏

Hello! thank you for the recommendations that I gratefully going to apply. Would you tell me what screenshot I should add to the readme file I have already 4. Also what framework I should mention, as far I know I don't use any?

**Review by Shaher-11 — APPROVED** [review]
Hello @eri8-9 ,

# Congratulations! 🎉 
#### Your project is approved!  There is nothing else to say other than... it's time to merge it :muscle: :muscle: 

## Optional Suggestions 💡
_The following point is  optional to implement however it's highly recommended!_:blush:

- Kidly have all the images in one folder as it's good practice to structure your projects well.

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Cheers and Happy coding!👏👏👏

**Pull request merged**